src/selectors/functions/builtInFunctions.fontoxpath.js

```diff
@@ -0,0 +1,59 @@
+import DynamicContext from '../DynamicContext';
+import Sequence from '../dataTypes/Sequence';
+
+/**
+ * @param  {../DynamicContext}      dynamicContext
+ * @param  {../dataTypes/Sequence}  query
+ * @param  {../dataTypes/Sequence}  args
+ */
+function fontoxpathEvaluate (dynamicContext, query, args) {
+	/**
+	 * @type {string}
+	 */
+	const queryString = query.first().value;
+	const variables = {};
+	if (args) {
+		args.first().keyValuePairs.reduce((expandedArgs, arg) => {
+			expandedArgs[arg.key.value] = arg.value;
+			return expandedArgs;
+		}, variables);
```

Comment:
Nah, we can always add more params. `fontoxpath:evaluate("...", map{"variables": map{"a": "b"}})` looks a bit 'much' IMHO